package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "ffc-ahwr-event",
-  "version": "0.8.5",
+  "version": "0.8.6",
   "description": "Azure Function to process Vet Vists back office events",
   "author": "Defra",
   "contributors": [
     "Steve Dickinson <steve.dickinson@defra.gov.uk"
   ],
+  "main": "ffc-ahwr-event/index.mjs",
   "scripts": {
     "lint": "standard --fix",
     "start": "func start -p 7080",
     "start:detached": "func start -p 7080 1> /dev/null 2> /dev/null &",
     "start:verbose": "func start -p 7080 --verbose",
-    "test": "jest",
-    "test:watch": "jest --coverage=false --onlyChanged --watch",
+    "test": "NODE_ENV=development jest",
```

Comment:
Isn't this normally 'test'? Maybe shouldn't be overriding here.

Reply:
Its set up so that it uses the development section of the babel config - we can change it to test if we feel like we should?

Reply:
I'd be surprised if there's not something in one of our environments relying upon it being 'test' by default.

Reply:
I think, as far as I know, we have a fair amount of isDev checks but very few isTest checks - I think the ones we do have are actually red herrings too.